roles/ceph-common/tasks/os_tuning.yml

```diff
@@ -22,4 +22,5 @@
     value={{ item.value }}
     state=present
     sysctl_file=/etc/sysctl.conf
+    ignoreerrors=yes
```

Comment:
FYI, `ignoreerrors` is a option for `sysctl`. Here is the [doc](http://docs.ansible.com/sysctl_module.html#options).